code/modules/reagents/chemistry/equilibrium.dm

```diff
@@ -379,10 +379,10 @@
 
 	//Apply thermal output of reaction to beaker
 	if(reaction.reaction_flags & REACTION_HEAT_ARBITARY)
-		holder.chem_temp += (reaction.thermic_constant* total_step_added*thermic_mod) //old method - for every bit added, the whole temperature is adjusted
+		holder.chem_temp += clamp((reaction.thermic_constant* total_step_added*thermic_mod), 0, 99999) //old method - for every bit added, the whole temperature is adjusted
```

Comment:
Where does 99999 come from? Can you make it a define, here and in 385?

Reply:
Arbitarily, but I made it a define